plugins/generator-1.20.4/neoforge-1.20.4/templates/potioneffect.java.ftl

```diff
@@ -78,22 +71,16 @@ public class ${name}MobEffect extends MobEffect {
 	</#if>
 
 	<#if hasProcedure(data.onActiveTick)>
-		@Override public void applyEffectTick(LivingEntity entity, int amplifier) {
-			<@procedureCode data.onActiveTick, {
-				"x": "entity.getX()",
-				"y": "entity.getY()",
-				"z": "entity.getZ()",
-				"world": "entity.level()",
-				"entity": "entity",
-				"amplifier": "amplifier"
-			}/>
+		@Override public boolean shouldApplyEffectTickThisTick(int duration, int amplifier) {
+			<#if hasProcedure(data.activeTickCondition)>

```

Comment:
activeTickCondition will now only work when onActiveTick is also applied which was not the case in previous generators